### ISSUE TITLE
Formatting description

### ISSUE THREAD

**Issue by kgrudzinski** [kgrudzinski/daily-log#4]
Do we want to add text formatting (markdown) for the description fields? In general, it would be nice thing to have, but

- It most likely means adding some kind of editor and/or preview. Technically, we can use current control (textarea) and write raw markdown, but it could be very annoying to fix formatting errors. We could write markdown in some other editor and copy it. Still annoying but would work as a temporary solution.
- Displaying should not be a problem, there are React libraries to render markdown, but iirc React does not like injecting random html in its dom tree.
- What formatting elements should be allowed? Images not. To much hassle. Lists: yes, tables: maybe. And what about links? If they are allowed, how do we handle opening them?

To summarize, it is nice feature to have, but it is not trivial and not very important.
